cubano-core/src/main/java/org/concordion/cubano/driver/action/DurationParser.java

```diff
@@ -0,0 +1,115 @@
+package org.concordion.cubano.driver.action;
+
+import java.time.Duration;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class DurationParser {
+
+    /**
+     * A string representation of this duration using ISO-8601 seconds based representation, such as PT8H6M12.345S.
+     * 
+     * The format of the returned string will be PTnHnMnS, where n is the relevant hours, minutes or seconds part of the duration. Any fractional seconds are placed after a decimal
```

Comment:
Isn't this going the other way around, and returning a description in the English language?

Reply:
Fixed